### DISCUSSION TITLE
Does acme.sh support cert issuance where the CA not the same as the DNS provider?

### DISCUSSION THREAD

**Discussion by WaitingForGuacamole** [acmesh-official/acme.sh#5118] [Q&A] [answered]
Let's say my domain is registered with GoDaddy, but I want to issue certs through NameCheap or some other non-GoDaddy provider. Is this possible, or do I need to use manual DNS validation?

**Comment by neilpang** [comment, marked as answer]
The CA and your DNS provider are unrelated. The CA (Let's Encrypt, ZeroSSL, ...) never talks to your registrar -- it just queries public DNS for the `_acme-challenge` TXT record. acme.sh only places that record wherever your domain's **authoritative DNS** lives.

So `--dns dns_<x>` depends on who hosts your nameservers, not your registrar or CA:

- Registered at GoDaddy, NS delegated to Cloudflare -> `--dns dns_cf`
- DNS served by GoDaddy -> `--dns dns_gd`

If your DNS host has no plugin, use DNS alias mode (CNAME `_acme-challenge` to a provider you can automate) or manual DNS validation.
